### PR TITLE
V.5 Release candidate

### LINKED ISSUE #14: Buzzer when battery is about to run out
Feature request: when the battery is low, an audible signal as well as the gauge would be helpful

### LINKED ISSUE #19: Move PAPRa firmware to designated repo
Personally, I would move the PAPRa firmware to a designated repo, as to provide clearer association of the codebase to its license and separation of version history between PCB & firmware code. The firmware repo can then be added as a submodule, if the PCB repo must reference the firmware - keeping in mind that submodules need to be updated.

### LINKED ISSUE #36: Bad fan boards can cook the m12 controller boards
When testing a "bad" fan board, multiple m12 v2 controller boards and a Milwaukee m12 charger were all rendered nonfunctional once power was applied to the system when testing a fan.

### LINKED ISSUE #46: Battery Circuit Board Assembly Diagram for May build
Need the assembly diagram for the board for the documentation

### LINKED ISSUE #47: test that protection circuitry actually protects
Going to also need to put that in the testing docs

### LINKED ISSUE #48: Test megaTinyCore v2.3.2
Looks like there is a new version of megaTinyCore, hopefully it resolves some of the timer issues @mechatroniks-git had with the now deprecated 2.3.1.

### PR DESCRIPTION
**Pull request by mechatroniks-git**
Closes #14
Closes #19
Closes #36
Closes #46
Closes #47
Closes #48

### CODE REVIEW

**Review by darraghbr — COMMENTED** [review]
Doesn't Drawings/D-TET-001102-2 Papra PCBA Drawing.pdf belong in the papr-print-files repo?

There also needs to be a drawing for just the PCBA on it's own rather than with the housing as well.

**Comment by mechatroniks-git** [comment]
> Doesn't Drawings/D-TET-001102-2 Papra PCBA Drawing.pdf belong in the papr-print-files repo?
> 
> There also needs to be a drawing for just the PCBA on it's own rather than with the housing as well.

@darraghbr Drawing updated, will move the assembly drawing to print files. Thanks for catching that.

File: firmware/papracode/LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (darraghbr) [inline firmware/papracode/LICENSE:1]: It doesn't look like we have a license file in the papra-pcb-firmware repo, if that isn't on purpose can we make an issue in that repo?

File: README.md [hunk 2]
```diff
@@ -6,22 +6,20 @@ Notes for programming:
...
-| ATtiny1614SSN_,| No          |    	                           |
-| ATtiny1624SSU, | No          |    	                           |
+| ATtiny214SSN_, | Yes         | See Below(1)  |
+| ATtiny414SSN_, | Yes         |    	                           |
+| ATtiny814SSN_, | Yes         |                                   |
+| ATtiny1614SSN_,| Yes         |    	                           |
+| ATtiny1624SSU, | Yes         |    	                           |
 
-ATtiny 204 and 214 must have all serial debug removed in addition to the startup sweeping LED sequence to fit in the 2K code space.
+(1)ATtiny 204 and 214 have 2K flash memory for program space. The preprocessor for TWOKFLASHCHIP must be uncommented prior to compiling. See Line 9.
```
Comment (darraghbr) [inline README.md:27]: Probably worth having at least some of this in the pcb-firmware repo?

File: firmware/papracode/LICENSE (hunk 1 above)
Comment (mechatroniks-git) [reply firmware/papracode/LICENSE:1]: Firmware is in a submodule, I'll create the license file issue over there.

File: README.md (hunk 2 above)
Comment (mechatroniks-git) [reply README.md]: @darraghbr READMEs have been adjusted.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+
 Notes for programming:
 * Make a UPDI Programmer: https://create.arduino.cc/projecthub/john-bradnam/create-your-own-updi-programmer-1e55f1?ref=user&ref_id=466812&offset=6
 * The board features a UPDI header on the top left of the PCB (top being the side with the battery connectors) the pins are top to bottom 5V, GND and data. 
 * Setup Arduino: https://www.hackster.io/john-bradnam/using-the-new-attiny-processors-with-arduino-ide-612185
 * You will need to use the megaTinyCore library by SpenceKonde: https://github.com/SpenceKonde/megaTinyCore
 
-![ArduinoConfig](https://user-images.githubusercontent.com/31858409/110745376-0f813100-8233-11eb-9dcd-ba6e8b78217b.PNG)
-
 Upload settings can be found in the arduino program.
```
Comment (darraghbr) [inline README.md:11]: Can you add this line onto the firmware repo and then we are good to go.

**Review by darraghbr — APPROVED** [review]

**Pull request merged**